packages/next/client/image.tsx

```diff
@@ -329,6 +329,20 @@ function handleLoading(
       onLoadingCompleteRef.current({ naturalWidth, naturalHeight })
     }
     if (process.env.NODE_ENV !== 'production') {
+      if (layout === 'raw') {
+        const heightModified =
+          img.height !== parseInt(img.getAttribute('height') || '')
+        const widthModified =
```

Comment:
Would it make more sense to skip the parse and do img.height.toString() != img.getAttribute(‘height’)

Reply:
Seems good. I changed it to that.